### ISSUE TITLE
Submit aria-label fix to blacklight-range-limit

### ISSUE THREAD

**Issue by andrewbenedictwallace** [UCLALibrary/ursus#221]
https://github.com/UCLALibrary/ursus/pull/220/files fixed an aria-label issue from blacklight by overriding a template locally. Ideally, this should be pushed upstream.

Acceptance
- [ ] A patch has been accepted into https://github.com/projectblacklight/blacklight_range_limit
- [ ] We are using a version that includes that patch
- [ ] We no longer use a local copy of app/views/blacklight_range_limit/_range_limit_panel.html.erb.
